spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/StaticGraph.scala

```diff
@@ -147,4 +153,59 @@ class StaticGraph[T: ClassTag](
     gradInput = fetchModelGradInput()
     gradInput
   }
+
+  private var inputFormats: Seq[Int] = null
```

Comment:
why seq instead of array ?

Reply:
inputFormats -> inputFormat ?

Reply:
As inputs nodes are seq, and inputFormats are corresponding to inputs nodes. 
So I use seq instead of array.

Reply:
I think "inputFormats" is more clear as there maybe multi inputs